Update AVDecVideoAcceleration_H264 to call out MF behavior

mattrwoz opened the pull request:
Add a clarifying statement to AVDecVideoAcceleration_H264 CodecAPI to describe MF behavior.  The current document is misleading in that it only applies when using DShow transforms.  But CodecAPIs are also available for MF transforms.

mattrwoz committed:
Update AVDecVideoAcceleration_H264 to call out MF behavior

PRMerger7 commented:
@mattrwoz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Update avdecvideoacceleration-h264-property.md

Thank you! Please see https://review.docs.microsoft.com/windows-authoring-guide/quickstart?branch=main for the private repos that we encourage MS folks to use.

-Steve

PRMerger3 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.
